views/index.erb

```diff
@@ -1,41 +1,150 @@
 <% if @profile %>
-  <h3><%= @profile["first_name"] %>'s Amazing Profile</h3>
+  <div class="section-one">
+  <div class="intro"><h3  class ="profile-value"><%= @profile["first_name"] %>'s Amazing Profile</h3></div>
+</div>
 
   <!-- TODO- Perhaps re-jig this to fit your new design / layout -->
-  <% @profile.each do |field, value| %>
+ <!--  <% @profile.each do |field, value| %>
     <div class="profile-section box">
-      <h4><%= field %></h4>
+      <h3><%= field %></h3>
       <p class="profile-value"><%= value %></p>
     </div>
-  <% end %>
+  <% end %> -->
 
-  <div id="stats" class="box">
-    <h3>Stats</h3>
-    <!-- TODO - Set the contents of this span with JavaScript -->
-    <span id="word-count">0</span> words found.
+  <div class="section-two">
+    <div class="title">
+      <h2><%=@profile.keys[0].capitalize%></h2>
```

Comment:
That's a nice and simple solution to the problem, great job! However, I think we could perhaps make this more dynamic. The approach you commented out would support any number of entries (key, value) in the `@profile` hash, whereas your approach is constrained by the reliance on knowing the specific index and reaching into the `keys` property. I see some common elements and repetition here that could be perfect candidates for a more 'dynamic' approach. What do you think?